### DISCUSSION TITLE
I build a figma unocss plugin, transform design to unocss

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [unocss/unocss#3392] [Show and tell]
Try it [figma to unocss](https://www.figma.com/community/plugin/1309119336695586856/to-unocss)

![iamge](https://github.com/zouhangwithsweet/figma-to-unocss/raw/main/public/demo.png)

**Comment by zyyv** [comment]
~~If #3314 is merged, you can send a PR to put it on the docs. ~~

**Reply by zouhangwithsweet** [reply]
Ok. Thanks.

**Reply by zyyv** [reply]
I take back what I said, feel free to send a PR mentioning your tool. 😄
